guides/user/kubernetes-v2/traffic-management/index.md

```diff
@@ -0,0 +1,270 @@
+automatically attaching a
+[Service](https://kubernetes.io/docs/concepts/services-networking/service/)
+to a workload during deployment, and defining pipelines to perform blue/green
+deployments.
+
+## Attaching a Service to a Workload
+
+Kubernetes manages the relationship between
+[Pods](https://kubernetes.io/docs/concepts/workloads/pods/pod/) and
+[Services](https://kubernetes.io/docs/concepts/services-networking/service/)
+using [Labels and
+Selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/).
+Spinnaker will acknowledge this relationship with the blue load balancer icon
+on a workload as shown here:
+
+{%
+  include
+  figure
+  image_path="./lb.png"
+%}
+
+You can additionally indicate to Spinnaker which Service objects a workload
+should be associated with for its lifetime using the
+[`traffic.spinnaker.io/load-balancers`](/reference/providers/kubernetes-v2/#traffic)
+annotation. To try this out, do the following:
+
+Deploy the following Service using the "Create Load Balancer" button:
+
+```yaml
+kind: Service
+apiVersion: v1
+metadata:
+  name: my-service
+spec:
+  selector:
+    app: myapp
+  ports:
+  - protocol: TCP
+    port: 80
+```
+
+{%
+  include
+  figure
+  image_path="./create-lb.png"
+%}
+
+And then deploy the following Replica Set using the "Create Server Group"
+button:
+
+```yaml
+apiVersion: apps/v1
+kind: ReplicaSet
+metadata:
+  name: frontend
+  labels:
+    tier: frontend
+  annotations:
+    traffic.spinnaker.io/load-balancers: '["service my-service"]' # <-- annotation
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      tier: frontend
+  template:
+    metadata:
+      labels:
+        tier: frontend
+    spec:
+      containers:
+      - name: frontend
+        image: gcr.io/google_samples/gb-frontend:v3
+```
+
+{%
+  include
+  figure
+  image_path="create-sg.png"
+%}
+
+Notice that the selector `app: myapp` appearing on the Service did not have to
+appear on the Replica Set's PodSpec, Spinnaker adds this automatically.
+
+### Caveats
+
+There are a few caveats when using this annotation:
+
+* The list of Services supplied in this annotation does _not_ need to specify
+  all Services a workload may be attached to. If your workload's labels already
+  attach it Service A, and you list Service B in the annotation, both Service A
+  and B will wind up serving traffic to this workload.
+
+  You can think of this annotation as only supplying the list of _additional_
+  services to attach to, not as an exhaustive list.
+
+* If you use this annotation, Spinnaker will edit the workload's labels, either
+  in the `metadata.labels`, or `spec.template.metadata.labels` sections. As a
+  result, if the labels added/removed to the `spec.template.metadata.labels`
+  conflicts with the label requirements in the object's
+  `spec.selector.match[Labels/Expressions]` section, the deployment will fail.
+
+* This annotation indicates to Disable and Enable stages which service objects
```

Comment:
Ha :P Let me describe it in more words and I bet you will have a better way to phrase it:

When you ask spinnaker to enable or disable a manifest, it will use this annotation to know which services the traffic changes are made with respect to. For example, if you specify service "A" in this annotation, even though you receive traffic from both "A" and "B" (due to how you set up your labels), and you request a "disable", only traffic from service "A" will be turned off.

Reply:
IIUC...

"This annotation tells Spinnaker which service or services to make traffic changes to. For example, if you specify service "A" in the annotation, only traffic from service "A" is disabled, even if you receive traffic from both services "A" and "B."

Reply:
Made a slight edit:

> This annotation tells Spinnaker which service or services to make traffic
  changes to. For example, if you specify service "A" in the annotation of a
  manifest and subsequently disable it, only traffic from service "A" will be
  disabled, even if you receive traffic from both services "A" and "B".